src/register-stories.js

```diff
@@ -0,0 +1,63 @@
+const Vue = require('vue')
+
+const capitalize = ([first, ...rest], lowerRest = false) =>
+  first.toUpperCase() + (lowerRest ? rest.join('').toLowerCase() : rest.join(''));
+
+const camelCase = str => {
+  let s =
+    str &&
+    str
+      .match(/[A-Z]{2,}(?=[A-Z][a-z]+[0-9]*|\b)|[A-Z]?[a-z]+[0-9]*|[A-Z]|[0-9]+/g)
+      .map(x => x.slice(0, 1).toUpperCase() + x.slice(1).toLowerCase())
+      .join('');
+  return s.slice(0, 1).toLowerCase() + s.slice(1);
+};
```

Comment:
I think it makes sense to use lodash for these instead of writing our own utilities, especially since this code will only be used in development so file size won't really matter.

Reply:
Agreed, I'll update this.